queue/abstract/driver/fifottl.lua

```diff
@@ -300,12 +300,18 @@ function method.release(self, id, opts)
     if task == nil then
         return
     end
+    if opts.touch_ttl == nil then opts.touch_ttl = true end
```

Comment:
It would be good to support it in 'utubettl' driver too.

Reply:
Added in new commit. Thanks for Your idea